src/arch.rs

```diff
@@ -159,6 +159,37 @@ pub trait Arch {
         None
     }
 
+    /// (optional) (LLDB extension) Return a `CallbackToken` that confirms that
+    /// the arch's register info of a single register has been send.
+    ///
+    /// Implementing this method enables LLDB to dynamically query the
+    /// target's register information one by one.
+
```

Comment:
nit: for consistency with the rest of the codebase, include empty `///` lines between blocks of comments